addons/rook/1.0.4-14.2.21/cluster/tmpl-ceph-storage-class.yaml

```diff
@@ -4,7 +4,7 @@ kind: StorageClass
 metadata:
   name: ${STORAGE_CLASS:-default}
   annotations:
-    storageclass.kubernetes.io/is-default-class: '"true"'
+    storageclass.kubernetes.io/is-default-class: "true"
```

Comment:
Are you sure this doesn't break after templating?

IIRC that's why it was double wrapped

Reply:
nvm I see the change to use _2